### DISCUSSION TITLE
Viewing and reporting annotations?

### DISCUSSION THREAD

**Discussion by apkawel** [GothenburgBitFactory/timewarrior#443] [Q&A]
Is there an easy way to generate a list of intervals that also shows their annotations? I undestand how to add annotations, but I don't know how to view/retrieve them. Thanks!

**Comment by sruffell** [comment]
Hmm...this doesn't' appear to be in any of the man pages that I see in the code. ~~I'll need to create an issue for it~~ Created #444 to address this.

That said, you can see part of the annotation with the `:annotations` hint to summary. For example:

```
timewarrior$ src/timew
Tracking proja
  Started 2021-08-03T23:00:10
  Current               07:12
  Total               0:07:02
timewarrior$ src/timew annotate @1 "This is a test"
Annotated @1 with "This is a test"
timewarrior$ src/timew summary :annotations

Wk  Date       Day Tags  Annotation        Start End    Time   Total
W31 2021-08-03 Tue proja This is a test 23:00:10   - 0:07:15 0:07:15
                                                                    
                                                             0:07:15
```

But the full annotation can currently only be seen with the export command, but that command can take a filter if you would like to limit the set of intervals exported.  For example:

```
timewarrior$ src/timew export :week
[
{"id":1,"start":"20210804T040010Z","tags":["proja"],"annotation":"This is a test"}
]
```
